cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/detector/SlowBrokerFinderTest.java

```diff
@@ -49,21 +56,28 @@ public void testDetectingSlowBrokerFromHistory() {
 
   /**
    * Test slow broker finder can detect broker which has consistently abnormally high metric in the cluster.
+   * The metric values used in test case is extracted from one incident we encountered in production.
    */
   @Test
   public void testDetectingSlowBrokerFromPeer() {
     SlowBrokerFinder slowBrokerFinder = createSlowBrokerFinder();
     Map<BrokerEntity, ValuesAndExtrapolations> currentMetrics = new HashMap<>(BROKER_ENTITIES.size());
     Map<BrokerEntity, ValuesAndExtrapolations> history = new HashMap<>(BROKER_ENTITIES.size());
-    currentMetrics.putAll(createCurrentMetrics(populateMetricValues(NORMAL_BYTES_IN_RATE, NORMAL_BYTES_IN_RATE, NORMAL_LOG_FLUSH_TIME_MS * 11),
-                          11, BROKER_ENTITIES.get(0)));
-    history.putAll(createHistory(populateMetricValues(NORMAL_BYTES_IN_RATE, NORMAL_BYTES_IN_RATE, NORMAL_LOG_FLUSH_TIME_MS * 11),
-                   10, BROKER_ENTITIES.get(0)));
+    currentMetrics.putAll(createCurrentMetrics(populateMetricValues(NORMAL_BYTES_IN_RATE, NORMAL_BYTES_IN_RATE, NORMAL_LOG_FLUSH_TIME_MS * 4),
+                                               11, BROKER_ENTITIES.get(0)));
+    history.putAll(createHistory(populateMetricValues(NORMAL_BYTES_IN_RATE, NORMAL_BYTES_IN_RATE, NORMAL_LOG_FLUSH_TIME_MS * 4),
```

Comment:
This test contains several hardcoded values, whose meanings are hard to infer. Is it possible to move them to static variables with clear names?